graphql.js

```diff
@@ -191,8 +203,7 @@
   * {a: {b: {c: 1, d: 2}}}, "a.b.c" => 1
   */
   GraphQLClient.prototype.fragmentPath = function (fragments, path) {
-    var getter = new Function("fragments", "return fragments." + path.replace(/\./g, FRAGMENT_SEPERATOR))
-    var obj = getter(fragments)
+    var obj = fragments[path.replace(/\./g, FRAGMENT_SEPERATOR)]
```

Comment:
why this is changed?

Reply:
Because it's equivalent, safer, and mentioned in #49